src/app/backend/resource/pod/poddetail.go

```diff
@@ -45,13 +51,26 @@ type PodDetail struct {
 	// Count of containers restarts.
 	RestartCount int32 `json:"restartCount"`
 
+	// Reference to the Creator
+	Creator Creator `json:"creator"`
+
 	// List of container of this pod.
 	Containers []Container `json:"containers"`
 
 	// Metrics collected for this resource
 	Metrics []metric.Metric `json:"metrics"`
 }
 
+// Creator is a view of the creator of a given pod, in List for for ease of use
+// in the frontend logic.
+//
+// Has 'oneof' semantics on the non-Kind fields decided by which Kind is there
+type Creator struct {
+	Kind string `json:"kind"`
+
+	JobList *joblist.JobList `json:"joblist,omitempty"`
```

Comment:
Ok, if this makes implementation easier, then good. 

@maciaszczykm Can you chime in here re showing this as a list row vs full details? I'm for showing this as a list row because all we want to do is to link to the creator, not show all details, since the user is looking at a pod here, not the creator.